modules/influxdb/build.gradle

```diff
@@ -3,6 +3,9 @@ description = "Testcontainers :: InfluxDB"
 dependencies {
     api project(':testcontainers')
 
-    compileOnly 'org.influxdb:influxdb-java:2.21'
-    testImplementation 'org.influxdb:influxdb-java:2.21'
+    implementation "org.influxdb:influxdb-java:2.21"
+    implementation "org.influxdb:influxdb-java:2.21"
```

Comment:
Duplicate:
```suggestion
    implementation "org.influxdb:influxdb-java:2.21"
```

Reply:
Also, previously it was `compileOnly`. @rnorth do we really need to make it a mandatory dependency?

Reply:
The other one should be `testImplementation`.

Reply:
Actually that's a good point. In the docs we're clear that the user must provide a driver, so `implementation` probably doesn't bring much value.